File: modul5/Python/PRAK501-2310817210023-RifkyPutraMahardika.py
def MaxBilangan(a,b,c,d):
    nilaimax1 = max(a,b)
    nilaimax2 = max(c,d)
    return max(nilaimax1,nilaimax2)

a,b,c,d = map(int, input().split())
hasil = MaxBilangan(a,b,c,d)
print(hasil)

File: modul5/C/PRAK501-2310817210023-RifkyPutraMahardika.c
#include <stdio.h>
int MaxBilangan(int a, int b, int c, int d) {
    int nilaimax1 = (a>b)? a:b; 
    int nilaimax2 = (c>d)? c:d;
    return (nilaimax1>nilaimax2)?nilaimax1:nilaimax2;
    }
int main() {
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}